components/lib/crash/metrics.yaml

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@mdboom does this file need to added to the probe-scraper too?

Reply:
Yes, it will need to be added.

Reply:
There's not much risk until someone actually integrates it into another app.  This is just another service that can handle crashes and until Fenix or some other app integrates it, it's harmless.  At this point in the game, I wasn't really thinking it would make it into Fenix, especially since this still requires data review, and would still need added to Fenix and pass review there.

Reply:
Right, perfect then :)